src/ontology/mondo-edit.obo

```diff
@@ -499803,8 +499806,8 @@ xref: Orphanet:611216 {source="MONDO:equivalentTo"}
 xref: UMLS:C5436906 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1754257"}
 is_a: MONDO:0000159 {source="OMIM:619151"} ! bone marrow failure syndrome
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/253 {source="OMIM:619151"} ! ADH5
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/404 {source="OMIM:619151"} ! ALDH2
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/253 ! ADH5
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/404 ! ALDH2
```

Comment:
these should not have been removed and are in the mondo-omim-genes.robot.tsv file